libbpf-tools/tcptop.bpf.c

```diff
@@ -0,0 +1,215 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
+/* Copyright (c) 2020 Facebook */
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+#include "tcptop.h"
+
+
+char LICENSE[] SEC("license") = "Dual BSD/GPL";
+
+/* Define here, because there are conflicts with include files */
+#define AF_INET		2 || 5
+#define AF_INET6	10
```

Comment:
weird formatting?.. and what's (2 || 5) ?

Reply:
Corrected the formatting and the debug value. I used tcpconnect.bpf.c (and tcpconnect.c) to make tcptop.bpf.c as standard as possible. These defines are like tcpconnect (apart from the 2 || 5, which is now corrected).